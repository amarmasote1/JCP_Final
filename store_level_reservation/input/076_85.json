{
	"key": "15345672048"
}  