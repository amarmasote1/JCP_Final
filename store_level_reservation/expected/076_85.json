{"status":200,"message":"Success","key":"ItemInventoryKey(itemID=15345672048)","body":{"itemID":"15345672048","onhandsupply":100,"shipATP":0,"bopusATP":0,"reservedDemand":0,"unconfirmedDemand":0,"nonStoreQualStatus":0,"networkQualStatus":0,"nonStoreOnhandQty":0,"futureAvlQty":0,"firstFutureAvailableDate":0,"shipActive":1,"bopusActive":1,"purifyATPDone":false,"factoryShipLeadTimeQty":0,"factoryShipLeadTimeUOM":0,"factoryShipInventoryIndicator":false,"whseClass":3,"moq":null},"warnings":null}